6.4.0/aio/data/bin/ampache_cron.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+DOW=$(date +%u)
+HOUR=$(date +"%H")
+
+# catalog clean / add / gather art
+if [[ $HOUR = 22 ]]; then
+  /usr/bin/php /var/www/bin/cli run:updateCatalog -cag
+fi
+# optimize DB on sunday
+if [[ $HOUR = 01 ]] && [[ $DOW = 6 ]]; then
```

Comment:
Is Sunday 7?

Reply:
Yes... I don't known why, but it was as it is :D